topology.yaml

```diff
@@ -74,3 +74,7 @@ start args:
     --predictable:
         action: store_true
         help: If specified, attempt to expose container ports to the same port number on the host
+    --spark2-version:
+        default: 2_1-r1
```

Comment:
Perhaps we should use dots to separate major and minor version instead of underscores? I think we do that for Kafka and Kudu (e.g. `1.4` instead of `1_4`) so we should be consistent.

Reply:
Done.